### PR TITLE
Code review request

### PR DESCRIPTION
**Pull request by shjang7**
We mainly programmed for 'user', 'post', 'comment' migration tasks.
Please review our code. thank you.

### CODE REVIEW

File: app/models/post.rb [hunk 1]
```diff
@@ -0,0 +1,6 @@
+class Post < ApplicationRecord
+  belongs_to :user
+  has_many :comment
```
Comment (rvvergara) [inline app/models/post.rb]: The word `comment` should be in the plural:

```ruby
has_many :comments
```

Comment (wathigo) [reply app/models/post.rb]: Thank you @rvvergara. We Updated the Post model

**Comment by wathigo** [comment]
Thank you @rvvergara .We have implemented the changes required in the Post model

**Comment by rvvergara** [comment]
All is well now!!! Great job you guys!!

# :+1: 

You may now merge the changes into `master`

**Pull request merged**